CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+### Added
+- Python 3.x support
+
+## [1.1.0] - 2020-02-10
+### Added
+- PowerDNS API support (https://doc.powerdns.com/recursor/http-api/index.html).
```

Comment:
Minor suggestion to make this a markdown link:

`[PowerDNS API support](https://doc.powerdns.com/recursor/http-api/index.html).`

Reply:
Updated